Update language to "debug" (#1485)
Qiskit/documentation: docs/verify/index.mdx

---
title: Introduction to debugging tools
description: Introduction to testing and debugging quantum circuits in Qiskit.
---
# Introduction to debugging tools

You can test your quantum programs by running them on simulated devices and exploring their performance under realistic device noise models. This allows you to debug them before sending them to a physical system.

Quantum simulators can be used to help develop and test programs before fine-tuning them and sending them to quantum hardware. Local simulators can do this with good performance and efficiency.

> Because the cost of classically simulating quantum circuits scales exponentially with the number of qubits, circuits that are larger than 50 qubits or so generally cannot be directly debugged. For such circuits, you can:

Reviewer: I think we should just explicitly say "cannot be directly run on simulators"

> Because the cost of classically simulating quantum circuits scales exponentially with the number of qubits, circuits that are larger than 50 qubits or so generally cannot be directly debugged. For such circuits, you can:

Author: ```suggestion
Because the cost of classically simulating quantum circuits scales exponentially with the number of qubits, circuits that are larger than 50 qubits or so generally cannot run on simulators. For such circuits, you can:
```

- Test smaller versions of the circuits that can be simulated classically.

> - Modify the circuits so that they become classically simulable and test these modified circuits.

Author: ```suggestion
- Modify the circuits so that they become classically simulable and test these modified circuits (note that this may also provide an inaccurate result).
```

> - Modify the circuits so that they become classically simulable and test these modified circuits.

Reviewer: is it simulable or simulatable?

Reviewer: I think it would also be worth mentioning for the second bullet point that modifying the circuit can make it classically simulatble but also provide an inaccurate result

Author: Both words work, according to the interwebs. (I wish we had a subscription to the OED!!!)
